angrmanagement/__main__.py

```diff
@@ -9,6 +9,8 @@
 import platform
 import signal
 
+from . import __version__
```

Comment:
I personally prefer `from angrmanagement import __version__` instead of the relative import.

Reply:
I have no issues with relative imports from the same directory. I.e. one dot is fine with me. If you dislike it you can change it but it doesn't play as well with tooling I've found.